### ISSUE TITLE
Request for help

### ISSUE THREAD

**Issue by wookjin** [Xinyu-Yi/PIP#35]
![image](https://github.com/Xinyu-Yi/PIP/assets/30368065/326a6e3e-5f64-4849-8275-e4b07d3980b7)

I purchased a PN3 sensor and set the network address of calc broadcasting with AxisStudio version 2.11 as shown in the picture.

However, 'evt.event_type' is output as 256. This is AvatarUpdated=256.

Is there a way to receive data with SensorModulesUpdated=1024 instead of AvatarUpdated=256?

What version of AxisStudio are you using?

If you use a program other than AxisStudio, what would it be?

I'm trying to follow along, but it doesn't work, so I'm posting a question.

![image](https://github.com/Xinyu-Yi/PIP/assets/30368065/0da708ac-c36b-40af-a844-b3e4a5e4a75f)
![image](https://github.com/Xinyu-Yi/PIP/assets/30368065/349e7ff9-1dfa-4ce9-a580-f105d5771e85)

**Comment by Xinyu-Yi** [comment]
I use Noitom Lab sensors (it contains 6 IMUs). I'm sorry but I'm not sure how to modify the code to fit AxisStudio. Maybe you need to refer to its api.

**Comment by wookjin** [comment]
Thank you very much for your reply.

"Run Noitom Axis Lab and send the real-time measurements to the udp port 7777." I'm not sure what the 'Notiom Axis Lab' program is in this sentence.

I thought the 'Noitom Axis Lab' program was the 'Axis Studio' program.

Where can I obtain the 'Noitom Axis Lab' program?

https://neuronmocap.com/

I couldn't find anything about "Noitom Axis Lab" on this site either.

**Comment by Xinyu-Yi** [comment]
No, I think the "Notiom Axis Lab" software is for "Axis Lab" sensors, while "AxisStudio" is for "PN3" sensors. Later I will send you the website for "Axis Lab" sensors here (as it is a new sensor, it is experimental and not listed on the official website currently). You may check if your PN3 sensors are compatible with the "Axis Lab" software (I am not sure). However, definitely, AxisStudio would work if you modify the code a bit using its api. Maybe try to ignore the AvatarUpdated event? Will your software also sends SensorModulesUpdated events?

**Comment by wookjin** [comment]
No, there doesn't seem to be a feature in the "Axis Studio" program that allows sending SensorModulesUpdated event data.

I hope to receive information about Noitom Lab soon.
